code/api/src/migrations/6-order.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('orders', {
```

Comment:
I see this is an example of creating a table, and below is the necessary inverted process.